packages/flutter/lib/src/material/pickers/calendar_date_picker.dart

```diff
@@ -668,14 +668,14 @@ class _MonthPickerState extends State<_MonthPicker> {
     });
   }
 
-  static const Map<TraversalDirection, Duration> _directionOffset = <TraversalDirection, Duration>{
-    TraversalDirection.up: Duration(days: -DateTime.daysPerWeek),
-    TraversalDirection.right: Duration(days: 1),
-    TraversalDirection.down: Duration(days: DateTime.daysPerWeek),
-    TraversalDirection.left: Duration(days: -1),
+  static const Map<TraversalDirection, int> _directionOffset = <TraversalDirection, int>{
```

Comment:
Looks like this PR also includes the changes from https://github.com/flutter/flutter/pull/60396. Is that intentional?

Reply:
Yeah, I am still waiting for the tree to go green for #60396 to land. When it does, I will merge it into this PR, and it won't show up as a diff.